modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -131,7 +131,7 @@ var backendServiceInvoker = function () {
         httpMethodObject.addRequestHeader(header);
         if (IS_OAUTH_ENABLED) {
             var accessToken = getAccessToken();
-            if (!(!accessToken.trim())) {
+            if (accessToken) {
```

Comment:
If "accessToken" is a boolean, use appropriate naming conventions recommended for the same. For instance, use something in the form of "isValidAccessToken" or (accessToken != null), if otherwise..